README.md

```diff
@@ -173,6 +173,13 @@ These are the Infrastructure Packages Gruntwork currently has available:
     1. [run-filebeat](https://github.com/gruntwork-io/package-elk/tree/master/modules/run-filebeat): Runs Filebeat on an application server to ship application logs off to Logstash cluster.
     1. [run-collectd](https://github.com/gruntwork-io/package-elk/tree/master/modules/run-collectd): Runs CollectD on an application server and ships off machine metrics to th Logstash cluster.
 
+1. **[Kubernetes / EKS](https://github.com/gruntwork-io/package-k8s)**: _(This package is in private beta)_ Deploy and manage a best practice EKS cluster. The main modules are:
```

Comment:
Well, `package-k8s` is a bit more general than that, right? "Deploy and manage a Kubernetes cluster and Kubernetes services."

Reply:
Yea I guess we should say it that way... I got too conservative since that's just the dream right now :)